keyboards/ymdk/oni_hhkb/keyboard.json

```diff
@@ -0,0 +1,236 @@
+{
+    "manufacturer": "YMDK",
+    "keyboard_name": "Oni HHKB",
```

Comment:
HHKB is a registered brand... even YMDK calls this "HHKB-style." The name "tsangan_oni" would make more sense here. "Tsangan layout" is more generally known and not a copyrighted name (as far as I know).